packages/core/src/signals/SignalContext.ts

```diff
@@ -49,6 +54,10 @@ export class SignalContext<
   TValue extends TSetterValue = TSetterValue,
   TOwner = void,
 > extends DependencyContext<TOwner> {
+  public static readonly DEFAULT = Symbol.for(
```

Comment:
As you've mentioned in your comment, I think `DEFAULT` should be exported as a separate symbol, not a class member.

Reply:
moved to `core/src/signals/symbols.ts`